src/transformers/models/trillsson_efficientnet/modeling_trillsson_efficientnet.py

```diff
@@ -0,0 +1,610 @@
+            be used for padding and conversion into a tensor of type *torch.FloatTensor*. See
+            [`TrillssonEfficientNetFeatureExtractor.__call__`] for details.
+        output_hidden_states (`bool`, *optional*):
+            Whether or not to return the hidden states of all layers. See `hidden_states` under returned tensors for
+            more detail.
+        return_dict (`bool`, *optional*):
+            Whether or not to return a [`~utils.ModelOutput`] instead of a plain tuple.
+"""
+
+
+@add_start_docstrings(
+    "The bare TrillssonEfficientNet model outputting raw hidden-states without any specific head on top.",
+    TRILLSSON_EFFICIENTNET_START_DOCSTRING,
+)
+class TrillssonEfficientNetModel(TrillssonEfficientNetPreTrainedModel):
+    def __init__(self, config: TrillssonEfficientNetConfig, add_pooling_layer: bool = True):
+        super().__init__(config)
+
+        self.config = config
+        # building first layer
+        input_channel = make_divisible(
+            24 * self.config.depth_multiplier, self.config.depth_divisible_by, self.config.min_depth
+        )
+
+        self.stem = TrillssonStem(
+            config=self.config,
+            in_channels=1,
+            out_channels=input_channel,
+            stride=2,
+        )
+
+        # building inverted residual blocks
+        self.block = nn.ModuleList()
+        current_num_blocks = 0
+        total_blocks = float(sum(blocks_args[0] for blocks_args in self.config.block_configs))
+        for expand_ratio, output_filters, num_repeat, stride, use_se, conv_type in self.config.block_configs:
+            output_channel = make_divisible(
+                output_filters * config.depth_multiplier, self.config.depth_divisible_by, self.config.min_depth
+            )
+            conv_block = TrillssonInvertedResidual if conv_type == 0 else TrillssonFusedInvertedResiduals
+            for i in range(num_repeat):
+                if i > 0:
+                    stride = 1
+                survival_probability = self.config.drop_connect_rate * current_num_blocks / total_blocks
+                self.block.append(
+                    conv_block(
+                        config=self.config,
+                        in_channels=input_channel,
+                        out_channels=output_channel,
+                        stride=stride,
+                        expand_ratio=expand_ratio,
+                        use_se=use_se,
+                        survival_probability=survival_probability,
+                    )
+                )
+                input_channel = output_channel
+                current_num_blocks += 1
+
+        # building last several layers
+        if config.depth_multiplier > 1.0:
+            output_channel = make_divisible(
+                1280 * config.depth_multiplier, config.depth_divisible_by, config.min_depth
+            )
+        else:
+            output_channel = 1280
```

Comment:
should the `1280` variable be in the config? Just wondering if we can not hard code it